File: Views/WebSources/js/model/Ticket.js
function Ticket  (
    id_ticket,
    titulo,
    descripcion,
    responsable,
    fecha_creacion,
    fecha_vencimiento,
    prioridad,
    estado,
    creador
){
    this.id_ticket = id_ticket;
    this.titulo = titulo;
    this.descripcion = descripcion;
    this.responsable = responsable;
    this.fecha_creacion = fecha_creacion;
    this.fecha_vencimiento = fecha_vencimiento;
    this.prioridad = prioridad;
    this.estado = estado;
    this.creador = creador;
}

File: Models/Ticket.cs
using IssueTracker_web_api.Models;
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Web;

namespace Issue_Tracker_Web_API.Models
{
    public class Ticket
    {
        public int Id { get; set; }
        public string Titulo {get; set; }
        public string Descripcion { get; set; }
        
        public string ResponsableId { get; set; }
        public DateTime FechaCreacion{ get; set; }
        public DateTime FechaFin { get; set; }
        public string Prioridad { get; set; }
        public string Estado { get; set; }
        
        public string CreadorId { get; set; }

        [ForeignKey("ResponsableId")]
        public ApplicationUser Responsable { get; set; }

        [ForeignKey("CreadorId")]
        public ApplicationUser Creador { get; set; }
    }
}